srcpkgs/droidcam/template

```diff
@@ -1,20 +1,21 @@
 # Template file for 'droidcam'
 pkgname=droidcam
-version=1.8.2
-revision=2
+version=2.1.3
+revision=1
 build_style=gnu-makefile
-make_build_args="USBMUXD=-lusbmuxd-2.0 JPEG=-lturbojpeg"
+make_use_env=true
+make_build_args="USBMUXD=-lusbmuxd-2.0 APPINDICATOR=ayatana-appindicator3-0.1"
 hostmakedepends="pkg-config"
 makedepends="pkg-config libjpeg-turbo-devel ffmpeg-devel alsa-lib-devel
  speex-devel libusbmuxd-devel libplist-devel gtk+3-devel
  libayatana-appindicator-devel"
-depends="v4l2loopback"
+depends="v4l2loopback linux-headers gcc make"
 short_desc="Tool for using your android device as a wireless/usb webcam"
-maintainer="ibrokemypie <ibrokemypie@bastardi.net>"
+maintainer="Erick Tucto <erick@ericktucto.com>"
```

Comment:
```suggestion
maintainer="Orphaned <orphan@voidlinux.org>"
```
while the current maintainer has not contributed in a while, we'd prefer adoptions only happen when the new maintainer has some history of contributions to void, to ensure the package gets the care it needs

https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#adopting-a-template

Reply:
@classabbyamp I'm sorry, I'll change the maintainer

Reply:
@classabbyamp I changed the maintainer